04_numeronym/nicholascross/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	nyms := []string{}
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices
Go style is to use 
`var nyms []string` for empty slices
or `nyms := make([]string, 0, len(vals))` if looking for optimisation